.formatter.exs

```diff
@@ -4,7 +4,21 @@ export_locals_without_parens = [
   plug: 2,
   forward: 2,
   forward: 3,
-  forward: 4
+  forward: 4,
+  match: 2,
+  match: 3,
+  get: 2,
+  get: 3,
+  post: 2,
+  post: 3,
+  put: 2,
+  put: 3,
+  patch: 2,
+  patch: 3,
+  delete: 2,
+  delete: 3,
+  options: 2,
+  options: 3
```

Comment:
Isn't it better to have a trailing comma so the removed line of `forward: 4` just to be replaced with `forward: 4,` does not happen in the future?

Reply:
I ran the code formatter on this file. Formatter doesn't do trailing commas